### PR TITLE
feat: Initial commit

### PR DESCRIPTION
**Pull request by kchernyshev**
Change-Id: I71f5140ef2b2a470aa9673262af1e7aa51b11972

BEGIN_COMMIT_OVERRIDE
feat: add chatmessagehistory and loader integrations
END_COMMIT_OVERRIDE

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+[project]
+name = "langchain-google-el-carro"
+dynamic = ["version"]
+description = "LangChain integrations for Google El Carro Oracle"
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.8"
+dependencies = [
+    "langchain>=0.1.8, <1.0.0",
```
Comment (averikitsch) [inline pyproject.toml:9]: nit: still needs to be removed

**Review by averikitsch — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (kchernyshev) [reply pyproject.toml:9]: we use it in test_load_and_spilt

Comment (averikitsch) [reply pyproject.toml:9]: Can we move it to tests dependencies?

Comment (kchernyshev) [reply pyproject.toml]: Done

**Pull request merged**